tools/jenkins/slaves/Dockerfile.rhel7

```diff
@@ -26,14 +26,16 @@ RUN set -x && \
     curl -sSL "$GECKODRIVER_DOWNLOAD_URL" | bsdtar -xvf - -C /usr/local/bin && \
     chmod +x /usr/local/bin/chromedriver /usr/local/bin/geckodriver && \
     yum install -y --setopt=tsflags=nodocs https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+    yum install -y --setopt=tsflags=nodocs https://cbs.centos.org/kojifiles/packages/git-crypt/0.6.0/1.el7/x86_64/git-crypt-0.6.0-1.el7.x86_64.rpm && \
     mkdir -p $HOME/bin
 
 # make sure context dir is at the repo root
 ADD . $HOME/verification-tests
 # RUN scl enable rh-git218 "git clone --depth 1 --single-branch 'https://github.com/openshift/verification-tests.git' '$HOME/verification-tests'"
 
 RUN scl enable rh-ruby26 "$HOME/verification-tests/tools/install_os_deps.sh"
-RUN scl enable rh-ruby26 "$HOME/verification-tests/tools/hack_bundle.rb"
+RUN scl enable rh-ruby26 "setpriv --reuid 1001 bundle config path $HOME/.gem/ruby"
+RUN scl enable rh-ruby26 "setpriv --reuid 1001 $HOME/verification-tests/tools/hack_bundle.rb"
```

Comment:
@akostadinov 
```
Step 7/14 : RUN scl enable rh-ruby26 "setpriv --reuid 1001 bundle config path $HOME/.gem/ruby"
 ---> Running in db7715a43c16

 ---> 19470f5ee9cd
Removing intermediate container db7715a43c16
Step 8/14 : RUN scl enable rh-ruby26 "setpriv --reuid 1001 $HOME/verification-tests/tools/hack_bundle.rb"
 ---> Running in 02cc209ea5c3

/home/jenkins/verification-tests/tools/hack_bundle.rb:25:in `write': Permission denied @ rb_sysopen - /home/jenkins/verification-tests/tools/hack_gemfiles/Gemfile_init.lock (Errno::EACCES)
	from /home/jenkins/verification-tests/tools/hack_bundle.rb:25:in `<main>'
The command '/bin/sh -c scl enable rh-ruby26 "setpriv --reuid 1001 $HOME/verification-tests/tools/hack_bundle.rb"' returned a non-zero code: 1
```

Reply:
Good catch, I didn't yet try to build and test these images. Thanks.